File: uri/1 - Basics/1036/Main.cpp
#include <cstdio>
#include <cmath>

int main(){
    double a, b, c;

    scanf("%lf %lf %lf", &a, &b, &c);

    if((pow(b,2) - 4*a*c) < 0 || 2*a == 0)
         printf("Impossivel calcular\n");
    else{
         printf("R1 = %.5lf\n",((-b) + sqrt((pow(b,2) - 4*a*c)))/(2*a));
         printf("R2 = %.5lf\n",((-b) - sqrt((pow(b,2) - 4*a*c)))/(2*a));
     }
    return 0;
}

File: uri/1 - Basics/1036/Main.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var a, b, c float64

	fmt.Scanf("%f %f %f", &a, &b, &c)

	if (math.Pow(b, 2)-4*a*c) < 0 || 2*a == 0 {
		fmt.Printf("Impossivel calcular\n")
	} else {
		fmt.Printf("R1 = %.5f\n", ((-b)+math.Sqrt((math.Pow(b, 2)-4*a*c)))/(2*a))
		fmt.Printf("R2 = %.5f\n", ((-b)-math.Sqrt((math.Pow(b, 2)-4*a*c)))/(2*a))
	}

}
